### DISCUSSION TITLE
Question about stable docker version with filament manager

### DISCUSSION THREAD

**Discussion by losless50** [OctoFarm/OctoFarm#1347] [Q&A]
Hey! quick question, since there's an issue with displaying the printer info, do you know if there's a earlier version of factotum I could use until this is issue is resolved? I would prefer to use docker-monolithic  but just a normal docker setup would work as well. Thanks so much for this project!

**Comment by NotExpectedYet** [comment]
My docker versionings are all messed up, monolithic is even worse. It's not had an image tag pushed since before 1.2.7 and same with the main version. 

I'm currently working on rectifying this over christmas with a new release mechanism and docker tags so sorry but until then or using < 1.2.7 on the main docker image is going to be the only option

If anyone knows docker and would like to help me out with my github action then feel free. The thing was donated to me and I don't understand it enough without tinkering more...

**Reply by losless50** [reply]
oh okay no problem! Totally understand!
